<launch>
  <arg name="config_file" />
  <node pkg="yac_ros" type="calib_mocap_node" name="calib_mocap_node" required="true" output="screen">
    <param name="config_file" value="$(arg config_file)" />
  </node>
</launch>
